packages/plugin-ext/src/main/browser/keybindings/keybindings-contribution-handler.ts

```diff
@@ -47,7 +47,16 @@ export class KeybindingsContributionPointHandler {
         if (!keybinding) {
             return undefined;
         }
-        const { command, when } = pluginKeybinding;
+        let { command, when } = pluginKeybinding;
```

Comment:
This is the part the keeps us from doing the whole conversion on the plugin side. @akosyakov I think this is an argument for moving the parsing of contributions to the plugin side. I think that would also reduce the amount of data to be transferred over the wire (re: your discussion with @benoitf)
But of course, that is out of scope for this PR

Reply:
Ideally the plugin system should register needed vscode commands and monaco commands without `monaco.` prefix in Theia command registry as well, otherwise we are not compatible with keybinding preferences for VS Code defined by users. If we have all vscode and monaco commands in our registry then mapping on the plugin side is not necessary. We need only apply generic conversion of positions/locations.

For performance one has to measure first whether there is an issue in this case.